### PR TITLE
🛂 Require recent authentication to view backup tokens

### PR DESCRIPTION
**Pull request by rowanseymour**
Captura de Pantalla 2021-02-23 a la(s) 16 33 13

### CODE REVIEW

File: temba/orgs/models.py [hunk 1]
```diff
@@ -67,13 +65,6 @@
 ORG_CREDITS_CACHE_TTL = 7 * 24 * 60 * 60  # 1 week
 
 
-@receiver(user_logged_in)
```
Comment (rowanseymour) [inline temba/orgs/models.py:70]: I added this before I had added an overridden login view.. now seems simpler to not use this and explicitly update `last_auth_on` from the views

**Review by ericnewcomer — APPROVED** [review]

**Comment by rowanseymour** [comment]
Have made some changes now so that the re-auth view extends the smartmin login view - it gives the correct handling of `next` redirect and should make it easier to also work in limiting of failed attempts using some of the existing FailedLogin logic - tho I need to make some changes in smartmin first

**Comment by rowanseymour** [comment]
Now updated to limit auth attempts

File: temba/tests/base.py [hunk 2]
```diff
@@ -154,6 +154,14 @@ def clear_storage(self):
         """
         shutil.rmtree("%s/%s" % (settings.MEDIA_ROOT, settings.STORAGE_ROOT_DIR), ignore_errors=True)
 
+    def login(self, user, update_last_auth_on: bool = True):
+        self.assertTrue(
+            self.client.login(username=user.username, password=user.username),
+            "Couldn't login as %(user)s:%(user)s" % dict(user=user.username),
```
Comment (ericnewcomer) [inline temba/tests/base.py:160]: This message looks funny

**Review by ericnewcomer — APPROVED** [review]
clever, i like it.

**Review by norkans7 — APPROVED** [review]

**Pull request merged**